xlml/apis/metric_config.py

```diff
@@ -101,4 +101,4 @@ class MetricConfig:
   json_lines: Optional[JSONLinesConfig] = None
   tensorboard_summary: Optional[SummaryConfig] = None
   profile: Optional[ProfileConfig] = None
-  use_runtime_generated_filename: bool = False
+  use_runtime_generated_gcs_folder: bool = False
```

Comment:
Let's remove this attribute to standardize the solution? So each test will have a dedicated folder for the output

Reply:
As dicsucced offline, we will remove this once all dags configs are changed to relative path. Added the TODO item.